pact/Pact/Core/PactValue.hs

```diff
@@ -109,6 +111,63 @@ instance Pretty PactValue where
       "CapToken" <> pretty (CapToken (fqnToQualName fqn) args)
     PTime t -> pretty (PactTime.formatTime "%Y-%m-%d %H:%M:%S%Q %Z" t)
 
+
+pactValueToText :: PactValue -> Text
+pactValueToText = \case
+  PLiteral lit -> case lit of
+    LString s -> tdquotes s
+    LInteger i -> tshow i
+    LDecimal d ->
+      if roundTo 0 d == d then
+        tshow (roundTo 0 d) <> ".0"
+      else tshow d
+    LUnit -> "()"
+    LBool b -> if b then "true" else "false"
+  PList l -> let
+    l' = pactValueToText <$> V.toList l
+    in tlist l'
+  PGuard g -> case g of
+    GKeyset (KeySet ks f) -> let
+      keys = tlist (fmap _pubKey $ S.toList ks)
+      p = predicateToText f
+      in tcurly "KeySet" [("keys", keys), ("pred", p)]
+    GKeySetRef ksn -> T.concat ["'", renderKeySetName ksn]
+    GUserGuard (UserGuard f args) -> let
+      f' = renderQualName f
+      args' = tlist (pactValueToText <$> args)
+      in tcurly "UserGuard" [("fun", f'), ("args", args')]
+    GCapabilityGuard (CapabilityGuard n args pid) -> let
+      mpid = [("pactId", maybe mempty _defPactId pid)]
+      pvs = pactValueToText <$> args
+      elems = [("name", renderQualName n), ("args", tlist pvs)] ++ mpid
+      in tcurly "CapabilityGuard" elems
+    GModuleGuard (ModuleGuard mn n) ->
+      tcurly "ModuleGuard" [("module", renderModuleName mn), ("name", n)]
+    GDefPactGuard (DefPactGuard pid n) ->
+      tcurly "PactGuard" [("pactId", _defPactId pid), ("name", n)]
+  PObject o -> let
+    o' = fmap (\(Field f, pv) -> T.concat [tdquotes f, ": ", pactValueToText pv]) $ M.toList o
+    in T.concat ["{",T.intercalate "," o', "}"]
+  PModRef (ModRef _mn imp) -> let
+    imp' = renderModuleName <$> S.toList imp
+    in T.concat ["module{",tlist imp', "}"]
```

Comment:
this is wrong. I fixed this in master. This should simply be `renderModuleName _mn`